readme.md

```diff
@@ -629,6 +629,7 @@
 - [Corda](https://github.com/chainstack/awesome-corda#readme) - Open source blockchain platform designed for business.
 - [Waves](https://github.com/msmolyakov/awesome-waves#readme) - Open source blockchain platform and development toolset for Web 3.0 apps and decentralized solutions.
 - [Substrate](https://github.com/substrate-developer-hub/awesome-substrate#readme) - Framework for writing scalable, upgradeable blockchains in Rust.
+- [Golem](https://github.com/golemfactory/awesome-golem#readme) - Open source peer-to-peer computing resources marketplace.
```

Comment:
I think it'd look better like this.
```suggestion
- [Golem](https://github.com/golemfactory/awesome-golem#readme) - Open source peer-to-peer marketplace computing resources.
```

Reply:
Thank you for taking a look and for the suggestion. I think this is less clear what it's describing unless you add a "for". So it'd need to be "Open source peer-to-peer marketplace **for** computing resources."

With the latter, it describes the same thing in more words but if you feel that's required to be clearer then I'm okay with it! :) Feel free to add "for" into the suggestion and I'll commit it.